frontendGraphApi/build.gradle

```diff
@@ -0,0 +1,11 @@
+dependencies {
+  vavr()
```

Comment:
Can be moved outside of `dependencies` section

Reply:
Ok, I'll do that myself on another PR to avoid another round of rebases ;)

Reply:
So no action required @mkondratek